janusgraph-core/src/main/java/org/janusgraph/util/system/ConfigurationLint.java

```diff
@@ -57,7 +60,12 @@ public static Status validate(String filename) throws IOException {
 
         final PropertiesConfiguration apc;
         try {
-            apc = new PropertiesConfiguration(filename);
+            FileBasedConfigurationBuilder<PropertiesConfiguration> builder =
+                new FileBasedConfigurationBuilder<PropertiesConfiguration>(PropertiesConfiguration.class)
+                    .configure(new Parameters().properties()
+                        .setFileName(filename)
+                        .setListDelimiterHandler(new DefaultListDelimiterHandler(',')));
+            apc = builder.getConfiguration();
```

Comment:
Similar or same code is duplicated in `CQLInputFormatIT.java`, `GraphApp.java`, `RemoteGraphApp.java`, `HBaseInputFormatIT.java`, `HBaseSnapshotInputFormatIT.java`, `TestGraphConfigs.java`, `JanusGraphFactory.java`, `ConfigurationLint.java`. I think it would make sense to move  `PropertiesConfiguration` creation logic to some util method or something like that. Basically, that we could simply execute `buildPropertiesConfiguration(String filename)` and it would return the instance of `PropertiesConfiguration`.